build-tools/packages/build-infrastructure/src/test/versions.vitest.ts

```diff
@@ -45,25 +44,34 @@ describe("setVersion", () => {
 		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 		await setVersion(main.packages, semver.parse("1.2.1")!);
 
-		const allCorrect = main.packages.every((pkg) => pkg.version === "1.2.1");
+		const results = main.packages.map((pkg) => {
+			return pick(pkg.packageJson, ["name", "version"]);
+		});
+
 		expect(main.version).to.equal("1.2.1");
-		expect(allCorrect).to.be.true;
+		expect(results).toMatchSnapshot();
```

Comment:
Can you include an example of what a snapshot match failure error looks like in the PR description?